packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -364,6 +366,28 @@ void main() {
         as Texture;
     expect(previewTexture.textureId, equals(textureId));
   });
+
+  test(
+      'takePicture binds and unbinds ImageCapture to lifecycle and makes call to take a picture',
```

Comment:
If you haven't already, it may be good to check with Xi whether this approach of binding/unbinding for each picture will have a significant performance cost.

I am currently doing the same thing in video capture and am also curious about the answer!